models/candidate.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+const reportSchema = require("./report").reportSchema;
+const Schema = mongoose.Schema;
+
+const candidateSchema = new Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    phone: {
+      type: String,
+      required: true,
+    },
+    dob: {
+      type: Date,
+      required: true,
+    },
+    location: {
```

Comment:
Where are the columns ssn, DL no? Check if all the data columns are present in the schema